app/src/main/java/com/jerboa/ui/components/common/ImageViewerDialog.kt

```diff
@@ -73,7 +73,6 @@ fun ImageViewerDialog(url: String, onBackRequest: () -> Unit) {
     }
 
     val backColor = MaterialTheme.colorScheme.scrim
-    val backColorTranslucent = MaterialTheme.colorScheme.scrim.copy(alpha = 0.4f)
```

Comment:
I actually do really like the background translucence, but I agree 40% in way too little... it should be like 10-20% . Could you try that out and see how it looks?